serverless.yml

```diff
@@ -13,6 +13,7 @@ provider:
   runtime: python3.8
   profile: serverless
   region: ${env:AWS_DEFAULT_REGION}
+  lambdaHashingVersion: 20201221
```

Comment:
What's this for?

Reply:
I saw this in warnings when deploying serverless.
It's a new feature of serverless that we need this line to already enable.

Reply:
It's a little improvement that will be default in version 3. So it's a good ideia to use now to avoid future problems

https://www.serverless.com/framework/docs/deprecations#default-providerlambdahashingversion